js/map.js

```diff
@@ -41,4 +51,48 @@ var GeoCodeAddress = function() {
       alert('Geocode was not successful for the following reason: ' + status);
     }
   });
-}
\ No newline at end of file
+}
+
+var DrawAllPrecincts = function() {
+  var url = "https://xv1zwa82p7.execute-api.us-east-1.amazonaws.com/dev/precincts";
+
+  globals.GetMap().data.loadGeoJson(url);
+  //Set any precinct that has no PCO to red
+  globals.GetMap().data.setStyle(function(feature) {
+    var pco = feature.getProperty('pco');
+    var color = pco ? 'white' : 'red';
+    return {
+      fillColor: color,
+    };
+  });
+
+  //if user clicks, show PCO info
+  globals.GetMap().data.addListener('click', function(event){
+    CreateInfoWindow(event.feature);
+    var anchor = new google.maps.MVCObject();
+    anchor.set("position", event.latLng)
+    globals.GetInfoWindow().open(globals.GetMap(), anchor);
+  });
+}
+
+//Change this to return new info window
+var CreateInfoWindow = function(feature) {
+    var pcoContent = '<div id="content">'
+      + '<h3>Precinct: ' + feature.getProperty('name') + '</h3>';
+    var pcoInfo = feature.getProperty('pco');
+
+    if(pcoInfo) {
+      pcoContent = pcoContent  +  
+      '<h5>' + pcoInfo['first'] + ' ' + pcoInfo['last'] + '</h5>';
+    } else {
+      pcoContent = pcoContent + '<p> No PCO currently.</p>' + 
+        '<a href="http://www.34dems.org/get-active/pco/"> Click here to be appointed!</a>';
```

Comment:
We're trying to not do that unless it redirects away from the site. It makes it less accessible for visually impaired visitors.